cpp/include/graphar/util/reader_util.h

```diff
@@ -25,10 +25,7 @@
 #include <vector>
 
 #include "graphar/fwd.h"
-
-namespace graphar {
-
-namespace util {
+namespace graphar::util {
```

Comment:
here should have an empty line between header include and the implementation code.